### ISSUE TITLE
Cinnamon plugin not working on k8 cluster

### ISSUE THREAD

**Issue by omnipresent07** [cfpb/hmda-platform#2570]
```
Error opening zip file or JAR manifest missing : /opt/docker/bin/../cinnamon/cinnamon-agent-2.10.13.jar
Error occurred during initialization of VM
agent library failed to init: instrument
```

**Comment by omnipresent07** [comment]
Could be related to #2571 . Will look at it once PR for that issue is merged.

**Comment by omnipresent07** [comment]
Related to `universalMappings` https://github.com/cfpb/hmda-platform/blob/master/commercial-build.txt#L70 which could be filtering all jars prior to creating docker image. Investigating further.

**Comment by kgudel** [comment]
Closed in #2577
